examples/all-clusters-minimal-app/all-clusters-common/all-clusters-minimal-app.zap

```diff
@@ -13602,19 +13602,51 @@
               "storageOption": "RAM",
               "singleton": 0,
               "bounded": 0,
-              "defaultValue": "0x00",
+              "defaultValue": "0x08",
```

Comment:
This is not right either.  This should be type 0, which is the simplest type that only supports the Lift feature.

Reply:
gotcha